collections/_article/dev-snapshot-godot-4-2-dev-2.md

```diff
@@ -0,0 +1,123 @@
+---
+title: "Dev snapshot: Godot 4.2 dev 2"
+excerpt: "Closing the first month of development of Godot 4.2, the second development snapshots features both smaller improvements and bigger reworks."
+categories: ["pre-release"]
+author: Yuri Sizov
+image: /storage/blog/covers/dev-snapshot-godot-4-2-dev-2.webp
+image_caption_title: "Design visualization system"
+image_caption_description: "A VR tool by Dan Silden"
+date: 2023-07-28 16:00:00
+---
+
+Final days of July mark the end of the first month of the Godot 4.2 development cycle, and things are starting to pick up steam. With Godot contributors having a few weeks to agree on, work on, review, and approve some of the new features, bigger changes are going to appear in the upcoming dev snapshots.
+
+As always, we try to offer a mix of smaller improvements, bug fixes, and bigger reworks for the Godot community to test and provide feedback. Here are some of the highlights in 4.2 dev2, with a bigger list available [below](#whats-new):
```

Comment:
```suggestion
As always, we try to offer a mix of smaller improvements, bug fixes, and new features for the Godot community to test and provide feedback. Here are some of the highlights in 4.2 dev2, with a bigger list available [below](#whats-new):
```

I suggest changing "bigger reworks" to "new features". They have a different meaning, but both are accurate in this case. The reason I prefer to avoid "bigger reworks" is it may give the impression that we are breaking compatibility by reworking existing features. 

While it is true that we are reworking some existing features, we are trying to do so in a way that maintains compatibility and only improves user experience. 

Since we also have cool new features coming, I think we can just highlight those and avoid the can of worms

Reply:
I agree with this logic. But I would keep "bigger", as I wanted to imply that we aren't only doing some insignificant tweaks and tune-ups.

```suggestion
As always, we try to offer a mix of smaller improvements, bug fixes, and bigger features for the Godot community to test and provide feedback. Here are some of the highlights in 4.2 dev2, with a bigger list available [below](#whats-new):
```